app/src/main/java/com/r4/notifications/mirror/MainActivity.java

```diff
@@ -323,7 +326,8 @@ private void stopReplyListenerService() {
 
     /**
      * Test only
-     * handles an incoming intent and replies to the notification if its an intent from an reply to the notification
+     * handles an incoming intent and replies to the notification if its an intent from an reply to the notification.
+     * TODO: add the feature to replace the 'replied-to-notification' with a notification that says 'it was replied to'
      */
```

Comment:
it brok e :(
needs changing in how the test notification works and or how the handlereply gets the id for the replacement notification.
also handlereply intent might be better placed in another class since its now probably a release user test feature